packages/@react-stately/collections/src/types.ts

```diff
@@ -37,7 +37,8 @@ export interface Node<T> extends ItemStates {
   wrapper?: ((element: ReactElement) => ReactElement) | void,
   parentKey?: Key,
   prevKey?: Key,
-  nextKey?: Key
+  nextKey?: Key,
+  props?: any
```

Comment:
Same question as above regarding if we want users to be able to pass any props

Reply:
this is internal API change for building Node in CollectionBuilder, so it won't affect `<Item />`
  example:    `<Item wasd={} onPress={)}>Folder 1</Item>` will get error for `wasd` as unknown property

but I could restrict it to ItemProps interface? `props?: ItemProps<T>`

Reply:
Yeah it would be good to restrict it in my opinion, don't want to have too many `any` types floating around.